apps/test-tsx-app/utils/prici-runner.ts

```diff
@@ -5,7 +5,11 @@ import { initialize } from '@prici/sdk';
 
 const token = 'abcd';
 const tenantName = 'demo-tenant-' + Math.random();
+let adminUi = 'false';
 
+export function updateAdminUi(val: string) {
```

Comment:
That's affecting global testing environment.

Try a more "scoped" solution.
E.g.: Pass that value to "getPriciSdk" and then to the "runPriciInstance" function.

Reply:
This was actually my first solution :)
I wondered if to use a boolean and then convert it to string when I set it in env or use string all the way as in this committed solution.